fs/fs.go

```diff
@@ -49,14 +55,22 @@ type partition struct {
 
 type RealFsInfo struct {
 	partitions map[string]partition
+	labels     map[string]string
```

Comment:
nit: can document the field? It's a map from label to mount point of the filesystem that has that label?